### PR TITLE
Implied timescales plot

### PR DESCRIPTION
**Pull request by jeiros**
## Proposed Feature / Bug-Fix

Adding an implied timescales plot, as I suggested in [#89](https://github.com/msmexplorer/msmexplorer/issues/89)

## Example Figure
Plot a list of msm objects like this:
```python
plot_implied_timescales(msm_objs, color_palette=colors,
                        xlabel='Lag time (frames)',
                        ylabel='Implied Timescales ($ns$)')
```
its-ex

## PR Check-list

- [x] Example added
- [x] Tests added
- [x] Docs added
  - Isn't it updated from the docstring of the function?
- [x] Changelog updated

### CODE REVIEW

File: msmexplorer/plots/msm.py [hunk 1]
```diff
@@ -190,3 +191,70 @@ def plot_timescales(msm, n_timescales=None, error=None, sigma=2,
...
+            pp.fill_between(x=lag_times,
+                            y1=[ts - err for ts, err in zip(timescales, errors)],
+                            y2=[ts + err for ts, err in zip(timescales, errors)],
+                            color=color)
+    ax.set_yscale('log')
+    if xlabel:
+        ax.xaxis.set_label_text(xlabel)
+    if ylabel:
+        ax.yaxis.set_label_text(ylabel)
+    ax.set_yscale('log')
```
Comment (mpharrigan) [inline msmexplorer/plots/msm.py]: Do you need to do this twice?

File: docs/changelog.rst [hunk 2]
```diff
@@ -9,6 +9,9 @@ v1.0.1 (Development)
 New Features
 ~~~~~~~~~~~~
 
+- Added a new function `plot_implied_timescales`` that accepts a list of msm objects
```
Comment (mpharrigan) [inline docs/changelog.rst]: there's a missing opening backtick here that's messing with github's rst parsing

**Comment by mpharrigan** [comment]
I would lobby for a log-log plot but looks good!

File: docs/changelog.rst (hunk 2 above)
Comment (jeiros) [inline docs/changelog.rst]: My bad! I'll fix it now.

File: msmexplorer/plots/msm.py (hunk 1 above)
Comment (jeiros) [inline msmexplorer/plots/msm.py]: What is done twice? I adapted that bit from the [plot_timescales](https://github.com/msmexplorer/msmexplorer/blob/master/msmexplorer/plots/msm.py#L175-L181) function

**Comment by jeiros** [comment]
Do you mean having a log x axis? I could do that, but it's up to the user? Since it will depend on the lag times that he looks at.

**Comment by mpharrigan** [comment]
> Do you mean having a log x axis? I could do that, but it's up to the user? Since it will depend on the lag times that he looks at.

Yeah, your example and test both have loggy x values, and I think this is the appropriate way to approach the problem. We're generally interested in order of magnitude lag-time. 1ns vs 2ns is inconsequential but you want to compare to e.g. 200 ns

File: msmexplorer/plots/msm.py (hunk 1 above)
Comment (cxhernandez) [inline msmexplorer/plots/msm.py]: I think @mpharrigan is referring to L253 and L258 both setting the `y_scale` to log values.

**Comment by cxhernandez** [comment]
Thanks for this @jeiros! I think maybe allowing users to set the axes scaling as an option might be nice, but maybe have the default be log-log.

File: msmexplorer/plots/msm.py (hunk 1 above)
Comment (jeiros) [inline msmexplorer/plots/msm.py]: d'oh! didn't see that one 🙄

**Pull request merged**